test/test.js

```diff
@@ -475,16 +451,16 @@ describe('Routes', () => {
         }
       )
       assert(updateRequest.ok)
-      const res = await fetch(`${spark}/retrievals/${retrievalId}`)
+      const { retrievalResultId } = await updateRequest.json()
+      const res = await fetch(`${spark}/retrieval-results/${retrievalResultId}`)
       await assertResponseStatus(res, 200)
       const body = await res.json()
-      assert.strictEqual(body.id, retrievalId)
+      assert.strictEqual(body.id, retrievalResultId)
       assert.strictEqual(body.cid, cid)
       assert.strictEqual(body.providerAddress, providerAddress)
       assert.strictEqual(body.protocol, protocol)
       assert.strictEqual(body.sparkVersion, '1.2.3')
       assert.strictEqual(body.zinniaVersion, '2.3.4')
-      assert(body.createdAt)
```

Comment:
In the new data schema, we don't have "retrieval created at", as this information won't be available in the new SPARK checker version.

We can use `finishedAt` field instead - this requires updating all Grafana charts.

Thoughts?

Reply:
Sgtm!